### DISCUSSION TITLE
能出TizenOS版本吗？

### DISCUSSION THREAD

**Discussion by ajavax** [xfangfang/wiliwili#453] [Ideas]
三星电视目前大部分基于TizenOS，没有相应的串流软件，希望能移植到TizenOS上，万分感谢🙏。

**Comment by xfangfang** [comment]
这个项目不接受tv端的相关讨论

（而且为了适配这个平台专门买台三星电视，成本有点过于高了哈，或许可以自己尝试做一下移植）
